core/cli/src/lib.rs

```diff
@@ -673,8 +673,15 @@ where
 		}
 	}
 
-	fs::remove_dir_all(&db_path)?;
-	println!("{:?} removed.", &db_path);
+	if let Result::Err(err) = fs::remove_dir_all(&db_path) {
```

Comment:
I would like to have a match here and also please move the `Ok(())` into this match statement.